MobileBuy/sample/src/main/java/com/shopify/sample/view/cart/CartCheckoutView.java

```diff
@@ -84,21 +105,14 @@ public CartCheckoutView(@NonNull final Context context, @Nullable final Attribut
   }
 
   @Override public void showError(final long requestId, final Throwable t) {
-    //TODO show error message
+    //TODO log and show error message
```

Comment:
Is this a really old TODO comment? If we don't need this, might as well just delete the comment. If we do need it, can it go in this PR?